infra/README.md

```diff
@@ -0,0 +1,55 @@
+# Meshdb Environment Setup
+
+1. Configure a user for the [proxmox provider](https://registry.terraform.io/providers/Telmate/proxmox/latest/docs) and setup env vars.
+2. Setup tfvars + ssh keys
+3. Create the VMs that will host k3s
+```
+cd meshdb/infra/tf/
+terraform init
+terraform plan --var-file=your.tfvars
+terraform apply --var-file=your.tfvars
```

Comment:
So we gotta fill out the example.tfvars?

Reply:
You need to fill out the values in whatever var file you reference

Reply:
Can that be added/explained in the docs?

Reply:
Good call, updated step 2